contracts/AnchoredView/SymbolConfiguration.sol

```diff
@@ -0,0 +1,119 @@
+pragma solidity ^0.6.6;
+pragma experimental ABIEncoderV2;
+
+// verbose configuration of anchored view, transforming
+// address to symbols and such
+contract SymbolConfiguration {
+    /// @notice Handpicked key for USDC
+    address public constant usdcOracleKey = address(1);
+
+    /// @notice Handpicked key for DAI
+    address public constant daiOracleKey = address(2);
+
+    /// @notice Frozen SAI price in ETH
+    uint public saiPrice = 1e18;
+
+    /// @notice The CToken contracts addresses
+    struct CTokens {
+        address cEthAddress;
+        address cUsdcAddress;
+        address cDaiAddress;
+        address cRepAddress;
+        address cWbtcAddress;
+        address cBatAddress;
+        address cZrxAddress;
+        address cSaiAddress;
+        address cUsdtAddress;
+    }
+
+    /// @notice The binary representation for token symbols, used for string comparison
+    bytes32 constant symbolEth = keccak256(abi.encodePacked("ETH"));
+    bytes32 constant symbolUsdc = keccak256(abi.encodePacked("USDC"));
+    bytes32 constant symbolDai = keccak256(abi.encodePacked("DAI"));
+    bytes32 constant symbolRep = keccak256(abi.encodePacked("REP"));
+    bytes32 constant symbolWbtc = keccak256(abi.encodePacked("BTC"));
+    bytes32 constant symbolBat = keccak256(abi.encodePacked("BAT"));
+    bytes32 constant symbolZrx = keccak256(abi.encodePacked("ZRX"));
+    bytes32 constant symbolSai = keccak256(abi.encodePacked("SAI"));
+    bytes32 constant symbolUsdt = keccak256(abi.encodePacked("USDT"));
+
+    /// @notice Address of the cToken contracts
+    ///  @dev must be updated to list a new token
+    address public immutable cEthAddress;
+    address public immutable cUsdcAddress;
+    address public immutable cDaiAddress;
+    address public immutable cRepAddress;
+    address public immutable cWbtcAddress;
+    address public immutable cBatAddress;
+    address public immutable cZrxAddress;
+    address public immutable cSaiAddress;
+    address public immutable cUsdtAddress;
+
+    /**
+     * @param tokens_ The CTokens struct that contains addresses for CToken contracts
+     */
+    constructor(CTokens memory tokens_) public {
+        cEthAddress = tokens_.cEthAddress;
+        cUsdcAddress = tokens_.cUsdcAddress;
+        cDaiAddress = tokens_.cDaiAddress;
+        cRepAddress = tokens_.cRepAddress;
+        cWbtcAddress = tokens_.cWbtcAddress;
+        cBatAddress = tokens_.cBatAddress;
+        cZrxAddress = tokens_.cZrxAddress;
+        cSaiAddress = tokens_.cSaiAddress;
+        cUsdtAddress = tokens_.cUsdtAddress;
+    }
+
+    /**
+     * comptroller expects price to have 18 decimals,
+     * additionally upscaled by 1e18 - underlyingdecimals
+     * base decimals is 1e6, so start by addint twelve
+     */
+    function getAdditionalScale(address cToken) public view returns (uint256) {
```

Comment:
These are fine, though I still think it might be even easier to simply return a tuple and then we don't need to repeat this logic multiple times. Also, should this be able to get rid of the other additional scale and one-offs used in the anchor contract above? Like, it'd be cool if there were the (generally) only metadata function(s) and thus we could get rid of statements like: `if (cTokenAddress == cUsdcAddress || cTokenAddress == cUsdtAddress) {`